module-presentation/src/main/java/com/depromeet/memory/dto/response/MemoryUpdateResponse.java

```diff
@@ -0,0 +1,216 @@
+package com.depromeet.memory.dto.response;
+
+import com.depromeet.image.domain.Image;
+import com.depromeet.image.dto.response.ImageSimpleResponse;
+import com.depromeet.member.dto.response.MemberSimpleResponse;
+import com.depromeet.memory.domain.Memory;
+import com.depromeet.memory.domain.MemoryDetail;
+import com.depromeet.memory.domain.Stroke;
+import com.depromeet.pool.domain.Pool;
+import com.fasterxml.jackson.annotation.JsonFormat;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class MemoryUpdateResponse {
```

Comment:
```suggestion
public class MemoryUpdateReadResponse {
```
이름 바꾸는 게 어떨까요?? 수정에 성공한 응답과 헷갈릴 수도 있을 것 같아요!

Reply:
`MemoryReadUpdateResponse`로 통일하는 것도 좋구요